### DISCUSSION TITLE
https://greendolphindance.com/2023/10/24/%E6%9C%80%E8%BF%91%E4%B8%8B%E8%BD%BD%E7%9A%84%E5%B0%8F%E7%8E%A9%E6%84%8F-Visual-Studio-Code/

### DISCUSSION THREAD

**Discussion by giscus** [greendolphindance/greendolphindance.github.io#9] [General]
# https://greendolphindance.com/2023/10/24/%E6%9C%80%E8%BF%91%E4%B8%8B%E8%BD%BD%E7%9A%84%E5%B0%8F%E7%8E%A9%E6%84%8F-Visual-Studio-Code/

Back to the rough ground!

https://greendolphindance.com/2023/10/24/%E6%9C%80%E8%BF%91%E4%B8%8B%E8%BD%BD%E7%9A%84%E5%B0%8F%E7%8E%A9%E6%84%8F-Visual-Studio-Code/

**Comment by redsherry** [comment]
您学到哪一课了？感觉很快就赶不上您的进度了

**Reply by greendolphindance** [reply]
还在html！

**Reply by redsherry** [reply]
感觉马上css了！

**Reply by greendolphindance** [reply]
那没有！
